pkgs/by-name/st/stown/package.nix

```
@@ -0,0 +1,46 @@
+{
+  fetchFromGitHub,
+  lib,
+  pkgs,
+  python3,
+}:
+let
+  version = "1.2.0";
+in
+python3.pkgs.buildPythonApplication {
+  pname = "stown";
+  inherit version;
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "rseichter";
+    repo = "stown";
+    tag = version;
+    hash = "sha256-iHeqmlo7be28ISJfPZ7GZC2gj2VVgt20ORnfYVToo0A=";
+  };
+
+  nativeBuildInputs = [
+    python3.pkgs.setuptools
+    python3.pkgs.wheel
+  ];
+
+  nativeCheckInputs = [
+    pkgs.tree
+    python3.pkgs.unittestCheckHook
+  ];
+
+  unittestFlagsArray = [
+    "-s"
+    "tests"
+    "-v"
+  ];
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
Wide `with` scopes are considered an [anti-pattern](https://nix.dev/guides/best-practices#with-scopes).